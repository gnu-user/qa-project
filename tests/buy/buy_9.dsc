TITLE
    Ticket Purchase Number (more than 4, buy-standard)
DESCRIPTION
    Run the login transaction using a buy-standard account. Initiate the buy 
    transaction. When prompted, enter a title which exists. When prompted, 
    enter a number of tickets to purchase which is greater than 4. Verify that 
    the amount is not accepted, and an error occurs.
DEPENDENCIES
    buy_8
