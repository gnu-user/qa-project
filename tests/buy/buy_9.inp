login
buyer
buy
The Godfather
5