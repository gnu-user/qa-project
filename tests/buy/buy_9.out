Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title:
Enter the number of tickets to purchase: 
Invalid number of tickets, cannot purchase more than 4 tickets.